packages/cli/react-native.config.js

```diff
@@ -98,6 +98,10 @@ module.exports = {
           name: "--no-interactive",
           description: "Disables interactive mode.",
         },
+        {
+          name: "--id [string]",
+          description: "Specify which bundle configuration to use.",
```

Comment:
I think we should be clearer on the intent of this flag:
```suggestion
          description: "Specify which bundle configuration to use if server configuration is missing.",
```

The entry in README should also be updated.